src/catalog_api.cpp

```diff
@@ -125,11 +125,4 @@ vector<IRCAPISchema> IRCAPI::GetSchemas(ClientContext &context, IRCatalog &catal
 	return result;
 }
 
-static string json_to_string(yyjson_mut_doc *doc, yyjson_write_flag flags = YYJSON_WRITE_PRETTY) {
```

Comment:
`static` scope the variable to current translation unit, so it's actually not used anywhere

Reply:
```sh
/tmp/duckdb_iceberg/src/catalog_api.cpp:128:15: warning: ‘std::string duckdb::json_to_string(duckdb_yyjson::yyjson_mut_doc*, duckdb_yyjson::yyjson_write_flag)’ defined but not used [-Wunused-function]
  128 | static string json_to_string(yyjson_mut_doc *doc, yyjson_write_flag flags = YYJSON_WRITE_PRETTY) {
      |               ^~~~~~~~~~~~~~
```